src/models/polyclinic.go

```diff
@@ -0,0 +1,41 @@
+package models
+
+type Polyclinic struct {
+	ID   int `gorm:"primaryKey"`
+	Name string
+}
+
+type PolyclinicRequest struct {
+	Name string `json:"name"`
+}
+
+type PolyclinicResponse struct {
+	ID   int    `json:"id"`
+	Name string `json:"name"`
+}
+
+func MapToNewPolyclinic(request PolyclinicRequest) Polyclinic {
+	return Polyclinic{
+		Name: request.Name,
+	}
+}
+
+func MapToExistingPolyclinic(request PolyclinicRequest, id int) Polyclinic {
+	return Polyclinic{
+		ID:   id,
+		Name: request.Name,
+	}
+}
+
+func MapToPolyclinic(polyclinic Polyclinic) PolyclinicResponse {
```

Comment:
This function was created for?

Should explain to function name, such as `MapToPolyclinicResponse`.